h/static/scripts/test/root-thread-test.js

```diff
@@ -50,6 +50,8 @@ describe('rootThread', function () {
       removeSelectedAnnotation: sinon.stub(),
       addAnnotations: sinon.stub(),
       setCollapsed: sinon.stub(),
+      selectTab: sinon.stub(),
+      TAB_ANNOTATIONS: 'annotation',
```

Comment:
Rather than redefining the constant here I'd suggest importing the _actual_ value. What we should do in future is get ourselves to a state where we can import interfaces, constants etc. in tests without importing the implementation.